app/controllers/films_controller.rb

```diff
@@ -1,16 +1,43 @@
 require 'pry'
+require 'open-uri'
 class FilmsController < ApplicationController
+  def add_movie
+    movie_id = params[:movie_id]
+      movie = Film.create(
+        rapid_id: params['movie_id'],
```

Comment:
permit params can be use here to validate the inputs

```
movie = Film.create(permit_params)
```

private 

def permit_param
end